webpack/reducers/forms_reducer.js

```diff
@@ -0,0 +1,12 @@
+import {
+  FETCH_FORMS_FULFILLED
+} from '../actions/types';
+
+const initialState = null;
+
+export default function forms(state = initialState, action) {
+  if (action.type == FETCH_FORMS_FULFILLED) {
+    return [...action.payload.data];
```

Comment:
This will create an array of forms in the store. We want to have it be a hash where the key is the id of the form and the value is the form object. You can do this with `_.keyBy(action.payload.data,'id');`